pkg/sql/lease.go

```diff
@@ -795,6 +795,7 @@ func (t *tableState) upsertLocked(ctx context.Context, table *tableVersionState,
 	s := t.mu.active.find(table.Version)
 	if s == nil {
 		t.mu.active.insert(table)
+		log.VEventf(ctx, 0, "new lease: %s with %s", s, table)
```

Comment:
`0` is pretty high, is there a reason you chose it to be that? Also, I don't even see any other call site that uses 0. Does that work as intended?